### PR TITLE
fix: fix react-docgen processing non-component files

### PR DESCRIPTION
**Pull request by lightbringer1991**
Resolve the following (and similar) errors when generating docs.
```bash
Error with path "src/components/Popover/WithRef.jsx": Error: No suitable component definition found.
  Error: No suitable component definition found.
```

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/components/Popover/WithRef.jsx [hunk 1]
```diff
@@ -28,21 +28,25 @@ const WithRef = ({
...
+          wrapperStyles={wrapperStyles}
+          dts={dts}
+          title={title}
+          popoverContent={popoverContent}
+          arrowStyles={arrowStyles}
+          innerRef={popperRef}
+        />,
+        boundariesElement
+      )}
+    </React.Fragment>
```
Comment (lightbringer1991) [inline src/components/Popover/WithRef.jsx:49]: `react-docgen` cannot identify `ReactDOM.createPortal` as a valid component, therefore putting it into fragment helps the library to parse this file successfully.

File: scripts/generate-docs [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+FILE_TO_IGNORE=("index.spec.jsx" "constants.js" "styles.js" "styles.spec.js" "dataProcessor.js" "dataProcessor.spec.js" "mocks.js" "mocks.jsx")
```
Comment (veegandhi) [inline scripts/generate-docs:3]: should be FILE**S**

File: scripts/generate-docs [hunk 3]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+FILES_TO_IGNORE=("index.spec.jsx" "constants.js" "styles.js" "styles.spec.js" "dataProcessor.js" "dataProcessor.spec.js" "mocks.js" "mocks.jsx")
+DIRECTORIES_TO_IGNORE=("fastStatelessWrapper")
+
+COMMAND="react-docgen src/components -o www/containers/props.json --pretty --resolver findAllComponentDefinitions"
+
+for ignoreFile in ${FILES_TO_IGNORE[@]}; do
+  COMMAND="$COMMAND --exclude $ignoreFile"
```
Comment (bchew) [inline scripts/generate-docs:9]: Does `--exclude` and `--ignore` not allow for an array/multiple?

Comment (lightbringer1991) [reply scripts/generate-docs:9]: https://github.com/reactjs/react-docgen/issues/126
it uses `node-dir` to parse these options.
I couldn't find an example of an array option, but multiple `--exclude` works.

**Comment by pphminions** [comment]
Merging 🚀

**Pull request merged**